cumulusci/core/exceptions.py

```diff
@@ -70,6 +70,14 @@ class DependencyResolutionError(CumulusCIException):
 class ConfigError(CumulusCIException):
     """ Raised when a configuration enounters an error """
 
+    def __init__(self, message=None, **kwargs):
+        super(ConfigError, self).__init__(message)
+        self.config_name = kwargs.get("config_name", "")
```

Comment:
Why not put config_name=None in the method signature? Also is there a point to this if we're not including it in the stringification of the exception?